pynest/nest/server/hl_api_server.py

```diff
@@ -191,6 +191,8 @@ def func_wrapper(call, args, kwargs):
             return func(call, args, kwargs)
         except nest.kernel.NESTError as e:
             abort(Response(getattr(e, 'errormessage'), 400))
+        except TypeError as e:
+            abort(Response(str(e), 400))
         except Exception as e:
             abort(Response(str(e), 400))
```

Comment:
I have meanwhile tested the code more thoroughly locally and found that these `abort()`calls don't necessarily print the cause for the exception to the terminal. A `print(e.msg))` in front of each of the `abort()` calls should fix this.